README.md

```diff
@@ -49,7 +49,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   * [CSS](#css)
   * [JavaScript](#javascript)
   * [React](#react)
-- [WEB-3](#web-3)
+- [Web3](#web-3)
```

Comment:
make sure to change it to `[Web3](#web3)` so, that it can direct to the content listed below

Reply:
Thanks for the review, corrected.